src/apps/clients/metadata.clj

```diff
@@ -0,0 +1,92 @@
+(ns apps.clients.metadata
+  (:use [slingshot.slingshot :only [throw+]])
+  (:require [apps.persistence.categories :as db-categories]
+            [apps.util.config :as config]
+            [metadata-client.core :as metadata-client]))
+
+(defn filter-by-avus
+  [username app-ids avus]
+  (metadata-client/filter-by-avus (config/metadata-client) username ["app"] app-ids avus))
```

Comment:
This is almost too trivial to mention, but it might be helpful to define a variable to hold the target type (just to make sure that a mistyped `"app"` constant doesn't cause problems in the future).